arm9/src/scrcmd_3.c

```diff
@@ -0,0 +1,467 @@
+#include "global.h"
+
+#include "heap.h"
+#include "map_header.h"
+#include "party.h"
+#include "pokedex.h"
+#include "pokemon_storage_system.h"
+#include "save_vars_flags.h"
+#include "scrcmd.h"
+
+extern void *FieldSysGetAttrAddr(struct FieldSystem *, u8 idx);
+extern SaveFashionData *Save_FashionData_Get(SaveData *save);
+
+extern void sub_02082140(void *, u8);
+extern void sub_02082158(void *, u8);
+extern BOOL sub_0205ED0C(struct SaveVarsFlags *state);
+extern void *sub_02081214(void *);
+extern void sub_02082630(void *, SaveData *, u32, u32 *);
+extern void sub_020814E8(void *);
+extern void sub_0208201C(void *, u16, MessageFormat *, u16);
+extern void sub_02082034(void *, u16, MessageFormat *, u16);
+extern void sub_0208206C(void *, u16, MessageFormat *, u16);
+extern void sub_02081DC4(void *);
+extern void sub_02081EB8(void *);
+extern void sub_02080E0C(TaskManager *, void *);
+extern void sub_02082090(void *, MessageFormat *, u16);
+extern void sub_020820B8(void *, MessageFormat *, u16);
+extern void sub_020820DC(void *, MessageFormat *, u16);
+extern u16 sub_0208213C(void *);
+extern void sub_0208210C(void *, void *, u16);
+extern u16 sub_0208217C(void *);
+extern u16 sub_02082190(void *);
+extern u16 sub_020821B4(void *);
+extern u16 sub_020821C4(void *, u16);
+extern u16 sub_020821DC(void *, u16);
+extern u16 sub_020821F0(void *);
+extern void sub_02082214(void *, u16 *, u16 *, u16 *, u16 *);
+extern u16 sub_02082254(void *);
+extern void sub_02082284(void *, MessageFormat *, u16, s32);
+extern u16 sub_020822F0(void *);
+extern void sub_0208223C(void *);
+extern void sub_0208224C(void *);
+extern void sub_02082444(void *, u16);
+extern BOOL sub_02082508(void *);
+
+extern void ov05_021F4608(u32);
+extern void ov05_021F45E8(u32);
+
+extern void sub_020825A0(void *, u32 *, u32 *, u32 *, u32 *, u32 *);
+
+extern void ov06_0224D3E8(TaskManager *);
+
+typedef struct unk_scrcmd_3 {
```

Comment:
struct names should be PascalCase
struct should also be in a header file, not in the c file